apis/io-engine/protobuf/v1/host.proto

```diff
@@ -20,16 +20,15 @@ service HostRpc {
   rpc StatNvmeController (StatNvmeControllerRequest) returns (StatNvmeControllerResponse) {}
 }
 
-message MayastorFeatures {
+message BackCompatMayastorFeatures {
+  // NVMe ANA enablement status.
   bool asymmetricNamespaceAccess = 1;
```

Comment:
Why do we need separate `BackCompatMayastorFeatures` ?

Reply:
Just for back-compat for existing code checking the old features directly from HostInfo.

Reply:
I guess another option could be to add a get_info_v2 and use new types for the whole thing?